outfit.js

```diff
@@ -39,4 +41,6 @@ class Outfit {
   generateIdNum() {
     return Math.random().toString(36).slice(2, 8);
   }
+
+
```

Comment:
remove this extra space

Reply:
no problem! We can delete one of those spaces